modularity/Modularity.ipynb

```diff
@@ -0,0 +1,360 @@
+{
```

Comment:
<p>how are these two plots different?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neurodata/connectocross/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Hi, actually I have no idea why there are two plots. I think they are exactly the same.